lib/terminus.spec.js

```diff
@@ -96,47 +96,33 @@ describe('Terminus', () => {
     })
   })
 
-  it('runs onSigterm when getting the SIGTERM signal', () => {
-    try {
-      execFileSync('node', ['lib/standalone-tests/terminus.onsigterm.js'])
-    } catch (ex) {
-      expect(ex.stdout.toString().trim()).to.eql('on-sigterm-runs')
-      return
-    }
-
-    throw new Error('running the test should throw, as the exitcode is not 0')
+  it('runs onSignal when getting the SIGTERM signal', () => {
+    let result = spawnSync('node', ['lib/standalone-tests/terminus.onsigterm.js'])
+    expect(result.stdout.toString().trim()).to.eql('on-sigterm-runs')
   })
 
-  it('runs onShutdown after onSigterm', () => {
-    try {
-      execFileSync('node', ['lib/standalone-tests/terminus.onshutdown.sigterm.js'])
-    } catch (ex) {
-      expect(ex.stdout.toString().trim()).to.eql('on-sigterm-runs\non-shutdown-runs')
-      return
-    }
-
-    throw new Error('running the test should throw, as the exitcode is not 0')
+  it('runs onShutdown after SIGTERM onSignal', () => {
+    let result = spawnSync('node', ['lib/standalone-tests/terminus.onshutdown.sigterm.js'])
```

Comment:
can we switch to const to be consistent with the rest of the tests?

Reply:
I'll get them switched over